applications/IgaApplication/custom_elements/shell_5p_element.cpp

```diff
@@ -199,8 +177,8 @@ namespace Kratos
         rKin.dtd1 = prod(rVar.P, rKin.dtd1);
         rKin.dtd2 = prod(rVar.P, rKin.dtd2);
 
-        rKin.metricChange[0] = inner_prod(rKin.A1, rKin.dud1) + 0.5 * norm_2_square(rKin.dud1);
-        rKin.metricChange[1] = inner_prod(rKin.A2, rKin.dud2) + 0.5 * norm_2_square(rKin.dud2);
```

Comment:
Is this still an issue, or can this be rolled back? Maybe norm_2_square is faster.

Reply:
I quite remember that this was an issue in windows. But, I will give it a go.